skycatalogs/skyCatalogs.py

```diff
@@ -359,9 +359,10 @@ def __init__(self, config, mp=False, skycatalog_root=None, verbose=False,
             self.cat_cxt.register_source_type('diffsky_galaxy',
                                               object_class=DiffskyObject)
         if 'sso' in config['object_types']:
-            self.cat_cxt.register_source_type('sso',
-                                              object_class=SsoObject,
-                                              collection_class=SsoCollection)
+            if self._sso_sed_factory:
+                self.cat_cxt.register_source_type('sso',
```

Comment:
See my comment below in `sed_tools.py`.  Silently omitting SSOs here seems wrong.

Reply:
I'll come up with a suitable place to log a warning message.